src/main/scala/com.snowplowanalytics.iglu/ctl/Main.scala

```diff
@@ -55,20 +57,20 @@ object Main extends IOApp {
       case Right(Command.StaticGenerate(in, out, schema, own, size, jp, raw, noheader, f)) =>
         Generate.process(in, out, jp, raw, schema, size, noheader, f, own)
       case Right(Command.StaticPush(input, registryRoot, apikey, public, legacy)) =>
-        Push.process(input, registryRoot, apikey, public, legacy)
+        EitherT(simpleClient.use { client => Push.process(input, registryRoot, apikey, public, legacy, client).value })
```

Comment:
TODO - remove duplication here - initially I thought it was just push/pull but it's worth revisiting